### ISSUE TITLE
Clarification on ground truth surface

### ISSUE THREAD

**Issue by bongbonglemon** [shunsukesaito/PIFu#38]
You described the ground truth surface to be a "0.5 level set of a continuous 3D occupancy field". To my understanding, this sounds like the middle slice of the model you got from RenderPeople. If this is true, you would only have the ground truths for PIFu outputs with the same input z-depth. I seek your clarification on this matter.

I would like to additionally enquire if you have the code used to obtain the 3D occupancy field from the raw model files you got from RenderPeople.

I look forward to your response and learning from it. Thank you.

**Comment by shunsukesaito** [comment]
In the occupancy representation, 1 means inside and 0 means outside. For ground truth, the transition happens exactly on the surface mesh, making 0.5 level set identical to the input mesh at the infinite resolution. PIFu is trained to predict this occupancy label with a neural network and the predicted occupancy field allows us to reconstruct the entire 3D geometry not only for the visible side but also for the occluded regions.

The released code includes ground truth occupancy sampling from RenderPeople.
https://github.com/shunsukesaito/PIFu/blob/30b428ba74bd7743a17c19fa20f6bfd39b1de057/lib/data/TrainDataset.py#L247
